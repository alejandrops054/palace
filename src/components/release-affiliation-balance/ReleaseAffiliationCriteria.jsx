/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from "react";
import { Row, Col, Button, Label, Input, Container } from "reactstrap";

function ReleaseAffiliationCriteria() {
  return (
    <>
      <span>Criteria</span>
      <Row className="border">
        <Col className=" " lg={{ size: 2 }} md={{ size: 2 }}>
          <Label check>
            <Input type="radio" name="radio1" /> Application
          </Label>
        </Col>
        <Col className=" " lg={{ size: 2 }} md={{ size: 2 }}>
          <Label check>
            <Input type="radio" name="radio1" /> Source
          </Label>
        </Col>
        <Col className=" " lg={{ size: 2 }} md={{ size: 2 }}>
          <Label check>
            <Input type="radio" name="radio1" /> Excel File
          </Label>
        </Col>
      </Row>
    </>
  );
}

export default ReleaseAffiliationCriteria;
