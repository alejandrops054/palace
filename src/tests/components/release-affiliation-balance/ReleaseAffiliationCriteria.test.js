/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import ReleaseAffiliationCriteria from '../../../components/release-affiliation-balance/ReleaseAffiliationCriteria'

describe('ReleaseAffiliationCriteria component', () => {
  test('debe mostrar snapshot', () => {

    const wrapper = shallow(<ReleaseAffiliationCriteria />)
    expect(wrapper).toMatchSnapshot();
  })

})