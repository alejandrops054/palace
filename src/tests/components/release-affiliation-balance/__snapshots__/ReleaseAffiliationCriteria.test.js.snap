// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReleaseAffiliationCriteria component debe mostrar snapshot 1`] = `
<Fragment>
  <span>
    Criteria
  </span>
  <Row
    className="border"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      className=" "
      lg={
        Object {
          "size": 2,
        }
      }
      md={
        Object {
          "size": 2,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Label
        check={true}
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Input
          name="radio1"
          type="radio"
        />
         Application
      </Label>
    </Col>
    <Col
      className=" "
      lg={
        Object {
          "size": 2,
        }
      }
      md={
        Object {
          "size": 2,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Label
        check={true}
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Input
          name="radio1"
          type="radio"
        />
         Source
      </Label>
    </Col>
    <Col
      className=" "
      lg={
        Object {
          "size": 2,
        }
      }
      md={
        Object {
          "size": 2,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Label
        check={true}
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Input
          name="radio1"
          type="radio"
        />
         Excel File
      </Label>
    </Col>
  </Row>
</Fragment>
`;
